assets/js/blocks/attribute-filter/block.js

```diff
@@ -28,10 +28,55 @@ import { updateAttributeFilter } from '../../utils/attributes-query';
  */
 const AttributeFilterBlock = ( {
 	attributes: blockAttributes,
-	isPreview = false,
+	isEditor = false,
 } ) => {
-	const [ displayedOptions, setDisplayedOptions ] = useState( [] );
-	const attributeObject = getAttributeFromID( blockAttributes.attributeId );
+	/**
+	 * Get the label for an attribute term filter.
+	 */
+	const getLabel = useCallback(
+		( name, count ) => {
+			return (
+				<Fragment key="label">
```

Comment:
I know this was not introduced in this PR, but is the `key` needed here?

Reply:
Probably got added because I suggested it was needed in a separate review.  But it's not :) I've removed (coming in a push).